File: libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/SdkVersionChecker.java
package com.mapbox.services.android.navigation.v5.internal.navigation;

public class SdkVersionChecker {

  private final int currentSdkVersion;

  public SdkVersionChecker(int currentSdkVersion) {
    this.currentSdkVersion = currentSdkVersion;
  }

  public boolean isGreaterThan(int sdkCode) {
    return currentSdkVersion > sdkCode;
  }

  public boolean isEqualOrGreaterThan(int sdkCode) {
    return currentSdkVersion >= sdkCode;
  }
}

File: libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/SdkVersionChecker.kt
package com.mapbox.services.android.navigation.v5.internal.navigation

class SdkVersionChecker(private val currentSdkVersion: Int) {

    fun isGreaterThan(sdkCode: Int): Boolean =
        currentSdkVersion > sdkCode

    fun isEqualOrGreaterThan(sdkCode: Int): Boolean =
        currentSdkVersion >= sdkCode
}
